### ISSUE TITLE
Is it possible to search data using this library also?

### ISSUE THREAD

**Issue by windchime18** [devmaster10/mysql-aes-encrypt#2]

**Comment by mahendrgurjar** [comment]
Yes, you can search data with laravel eloquent query. It will add the decrypt method automatically. But searching will be case sensitive you have to use lower case or upper case in the query.
